### PR TITLE
Implemented: Support to store user preference for selected product store(#2f2h8hu)

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
- [ ] I have changed the target to hacktoberfest branch

### CODE REVIEW

**Review by disha1202 — CHANGES_REQUESTED** [review]
Pass token while dispatching getProfile.
https://github.com/hotwax/inventory-count/blob/main/src/store/modules/user/actions.ts#L36

https://github.com/hotwax/inventory-count/blob/main/src/store/modules/user/actions.ts#L50

File: src/store/modules/user/actions.ts [hunk 1]
```diff
@@ -80,16 +80,29 @@ const actions: ActionTree<UserState, RootState> = {
...
+  async getProfile ({ commit }, payload) {
+    const resp = await UserService.getProfile(payload)
+    if (resp.status === 200 && resp.data.facilities?.length > 0) {
       const localTimeZone = moment.tz.guess();
       if (resp.data.userTimeZone !== localTimeZone) {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
+
       commit(types.USER_INFO_UPDATED, resp.data);
       commit(types.USER_CURRENT_FACILITY_UPDATED, resp.data.facilities.length > 0 ? resp.data.facilities[0] : {});
```
Comment (disha1202) [inline src/store/modules/user/actions.ts:92]:
```suggestion
      dispatch("setFacility", { facility: resp.data.facilities[0] });
```

**Review by disha1202 — CHANGES_REQUESTED** [review]

File: src/store/modules/user/actions.ts [hunk 2]
```diff
@@ -106,13 +119,59 @@ const actions: ActionTree<UserState, RootState> = {
...
+        
+        let userPrefStore = ''
+
+        try {
+          const userPrefResponse =  await UserService.getUserPreference({
+           'userPrefTypeId': 'SELECTED_BRAND'
+          });
+          userPrefStore = user.stores.find((store: any) => store.productStoreId == userPrefResponse.data.userPrefValue)
+        } catch (err) {
+          console.log(err)
```
[the remark this answers is not in the document]
Comment (shashwatbangar) [reply src/store/modules/user/actions.ts]: Done

File: src/store/modules/user/actions.ts (hunk 1 above)
Comment (shashwatbangar) [reply src/store/modules/user/actions.ts]: Done

**Comment by dt2patel** [comment]
not using product store in inventory count app. closing PR

**Pull request closed without merging**